roles/variables/defaults/main.yml

```diff
@@ -728,7 +744,7 @@ kafka_broker_copy_files: []
 kafka_broker_default_internal_replication_factor: "{{ [ groups['kafka_broker'] | default(['localhost']) | length, default_internal_replication_factor ] | min }}"
 
 ### Boolean to enable the kafka's metrics reporter. Defaults to true if Control Center in inventory. Enable if you wish to have metrics reported to a centralized monitoring cluster.
-kafka_broker_metrics_reporter_enabled: "{{ confluent_server_enabled and 'control_center' in groups }}"
+kafka_broker_metrics_reporter_enabled: "{{ confluent_server_enabled and ('control_center' in groups or 'control_center_next_gen' in groups) }}"
```

Comment:
did we miss adding kafka_broker_metrics_reporter_for_control_center_next_gen_enabled here similar to the variable added for kraft?